01-Login/src/app/components/home-content/home-content.component.ts

```diff
@@ -1,15 +1,8 @@
 import { Component, OnInit } from '@angular/core';
-import { faLink } from '@fortawesome/free-solid-svg-icons';
```

Comment:
*Author note:* Using the implicit reference means that we can take the references to the icons out from each component entirely, cleaning up the code.